Polish Secrets Manager Doc (#332)
astronomer/docs: cloud/secrets-backend.md

- A [service account](https://cloud.google.com/iam/docs/creating-managing-service-accounts) with the [Secret Manager Secret Accessor](https://cloud.google.com/secret-manager/docs/access-control) role on Google Cloud.
- A [JSON service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating_service_account_keys) for the service account.

### Step 1: Write an Airflow Variable or Connection to Google Cloud Secret Manager

To start, add an Airflow variable or connection as a secret to Google Cloud Secret Manager. You can do so via the Cloud Console or the gcloud CLI.

Secrets must be formatted such that:
- Airflow variables are set as `airflow-variables-<variable-key>`.
- Airflow connections are set as `airflow-connections-<connection-id>`.

> For example, to add an Airflow variable whose key is `my-secret-variable`, you would run the following gcloud CLI command -

Reviewer: ```suggestion
For example, to add an Airflow variable with a key `my-secret-variable`, you would run the following gcloud CLI command:
```
